Dockerfile

```diff
@@ -21,7 +21,8 @@ ENV NDK_CCACHE /usr/bin/ccache
 # Keep the packages in alphabetical order to make it easy to avoid duplication
 RUN DEBIAN_FRONTEND=noninteractive dpkg --add-architecture i386 \
     && apt-get update -qq \
-    && apt-get install -y bsdmainutils \
+    && apt-get install -y apt-utils \
```

Comment:
But this the docker file for build, should have nothing to do with the ROS docker which is in `tools/sync_test_server/Dockerfile`. What is the error message when you call `start-server`?